reactful-numbers/src/app/components/Button.js

```diff
@@ -1,5 +1,9 @@
 import React from "react";
-function Button({ buttonColor, buttonText }) {
-  return <button className={`button ${buttonColor}`}>{buttonText}</button>;
+function Button({ buttonColor, buttonText, onClick }) {
```

Comment:
Passing the onClick function to the Button component from the ButtonBox component where the functionallity is.